tools/cmake/downloader.py

```diff
@@ -1,40 +1,42 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python3
 
 import sys
-import urllib2
+import urllib3
```

Comment:
Looks like this might not be available:

```
[ 18%] Downloading: https://s3.amazonaws.com/osquery-packages/third-party/pre-built/none/Jinja2-2.10-py2.py3-none-any.whl
Traceback (most recent call last):
  File "/__w/1/s/tools/cmake/downloader.py", line 4, in <module>
    import urllib3
ImportError: No module named urllib3
libs/fb/jinja2/CMakeFiles/thirdparty_pythonmodule_jinja2_downloader.dir/build.make:60: recipe for target 'libs/fb/jinja2/Jinja2-2.10-py2.py3-none-any.whl' failed
```

Reply:
Yeah, I was trying to fix this but can't yet figure how to add `urllib3` as a python dependencies.